README.md

```
@@ -1,144 +1,158 @@
+| `@paper-tools/drilldown` | Deep-dive search using keywords and venues via DBLP and Crossref |
+| `@paper-tools/visualizer` | CLI for visualizing OpenCitations citation graphs (outputs in JSON, DOT, or Mermaid) |
+| `@paper-tools/web` | Next.js Web UI for searching, visualizing, and recommending papers |
 
 ## Setup
 
-```bash
-pnpm install
-cp .env.example .env
-```
+1. Install dependencies:
+   ```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)